index.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var _ = require('lodash');
+var path = require('path');
+var fs = require('fs');
+var through2 = require('through2');
+var gettextParser = require('gettext-parser');
+var Handlebars = require('handlebars');
+
+
+var replaceText = function(catalog, opts, chunk, enc, callback) {
+  var template = _.template(chunk.toString(), {
```

Comment:
>I've learned the hard way that sending whole files to lodash template is terribly inefficient—if the file has any size, it will exceed node's default stack size (and then you have to have nasty helper scripts to bump node's stack size). I fully regret using it and advise not doing so here.

When we ran it, it executed much faster than Handlebars. Of course that doesn't say much about memory consumption. The reason we used lodash templates is that they allow us to use a custom interpolator, which is definitely necessary for consumers that want to bundle handlebars templates in their code which are not static.